pkg/controller/user/user_controller.go

```diff
@@ -17,7 +17,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-var log = logf.Log.WithName("controller_user")
+var (
+	log = logf.Log.WithName("controller_user")
+	// A set of pre configured groups used to exclude a user from rhmi specific groups
```

Comment:
Just leftover comment from moving the constants to `userHelper.go`. Otherwise looks good ! :+1: